### PR TITLE
[Wallet] Fix verification signing issue for Alfajores/Pilot

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Fix verification signing issue for Alfajores/Pilot
- Add a settings menu to go to verification if user skipped/failed 
- Related navigation changes

### Tested

Did verification successfully in pilot
Tested navigation changes

### Related issues

- #1523 

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/locales/es-419/nuxVerification2.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "getVerified": "~~Verify Phone Number",
```
Comment (i1skn) [inline packages/mobile/locales/es-419/nuxVerification2.json:2]: Just as a question, what `~~` is for?

**Review by i1skn — APPROVED** [review]
LGTM 👍

File: packages/mobile/locales/es-419/nuxVerification2.json (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/locales/es-419/nuxVerification2.json:2]: That's just a signal for me to know these are placeholders until the actual translations come in from our contractor

Comment (jmrossy) [reply packages/mobile/locales/es-419/nuxVerification2.json]: That's just a signal for me to know these are placeholders until the actual translations come in from our contractor

**Pull request merged**